packages/core/src/controllers/history.ts

```diff
@@ -80,6 +82,7 @@ export class JsonRpcHistory extends IJsonRpcHistory {
       topic,
       request: { method: request.method, params: request.params || null },
       chainId,
+      expiry: Date.now() + calcExpiry(THIRTY_DAYS),
```

Comment:
Don't need the explicit `Date.now()` call, handled by `calcExpiry` internally if not passed as second param: https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/utils/src/misc.ts#L313

```suggestion
      expiry: calcExpiry(THIRTY_DAYS),
```

Reply:
I tried it and it didn't calculate the correct expiry. It was calculating 30 days in milliseconds rather than `Date.now() + calcExpiry(THIRTY_DAYS)` may be bug in the util?

Reply:
the tests confirm the expected behaviour